src/NzbDrone.Core/Notifications/MailGun/MailGunPayload.cs

```diff
@@ -0,0 +1,10 @@
+namespace NzbDrone.Core.Notifications.MailGun
+{
+    public class MailGunPayload
```

Comment:
Empty payload?

Reply:
Work in progress as the NetworkCredential thing prevents me from testing it really before that's sorted.

Reply:
It seems that Mailgun requires the body to be sent as form-data, but I'm not sure if the httpClient currently supports that?

Reply:
`HttpClient` executes/processes a `HttpRequest`, should just be a matter of setting up the request properly. There is also `HttpRequestBuilder.ApplyFormData`.